nodebuilder/tests/swamp/config.go

```diff
@@ -15,7 +15,8 @@ type Components struct {
 // 100ms
 func DefaultComponents() *Components {
 	cfg := core.DefaultTestConfig()
-	cfg.Tendermint.Consensus.TimeoutCommit = 100 * time.Millisecond
+	// timeout commits faster than this tend to be flakier
+	cfg.Tendermint.Consensus.TimeoutCommit = 200 * time.Millisecond
```

Comment:
I've noticed that this makes the testnodes less flakey in the CI, could increase it even further if we continue to see the `temp directory not empty` bug in CI